### PR TITLE
feat(kibbeh): added language search

### LINKED ISSUE #2731: Cannot switch back to English after you change profile language.
**Describe the bug**

**To Reproduce**
1. Go to 'any page'
2. Click on 'profile/langauge then select a language'
3. .....
4. English is not listed as a language, so if you are really English speaking and click on another language you cannot change back to English.

**Expected behavior**
Should be able to change from English to another language and then back to English.

### PR DESCRIPTION
**Pull request by firecraftgaming**
closes #2731

### CODE REVIEW

**Comment by firecraftgaming** [comment]
image

File: kibbeh/package.json [hunk 1]
```diff
@@ -61,11 +61,13 @@
     "react-use-gesture": "^9.1.3",
     "react-virtual": "^2.7.1",
     "superstruct": "^0.15.1",
+    "tsc": "^2.0.3",
```
Comment (amitojsingh366) [inline kibbeh/package.json:64]: add this in dev deps

Comment (amitojsingh366) [inline kibbeh/package.json]: add this in dev deps

File: kibbeh/package.json [hunk 2]
```diff
@@ -61,11 +61,13 @@
...
     "tslib": "^2.2.0",
     "twemoji-parser": "^13.0.0",
     "use-debounce": "^6.0.1",
     "uuid": "^8.3.2",
     "wakelock-lazy-polyfill": "^1.0.0",
+    "yarn": "^1.22.10",
```
Comment (amitojsingh366) [inline kibbeh/package.json]: please don't keep this in the deps at all

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]

**Comment by amitojsingh366** [comment]
please follow the proper pattern for commit messages *(💡TIP: you can use `yarn commit` or `yarn commit:signed` and answer the questions asked to generate a properly formatted one for you!)*

to fix your most recent commit you can run the command `git commit --amend -m "properly formatted commit message here"`

to fix your older/multiple commits please take a look at [this article](https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message#amending-older-or-multiple-commit-messages)

File: kibbeh/src/ui/LanguageSearch.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export const LanguageSearch: React.FC<{ onChange: (value: string) => void }> = ({ onChange }: { onChange: (value: string) => void }) => {
+  const classes = `
+  flex w-full items-center px-4 py-4 md:py-2 cursor-pointer md:hover:bg-primary-700
```
Comment (amitojsingh366) [inline kibbeh/src/ui/LanguageSearch.tsx]: remove the `cursor-pointer` class

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]
also you made the search case sensitive, it shouldn't be like that. i want to be able to type in "english" to get the suggestion for English. rn i need to type the "E" capital

**Comment by amitojsingh366** [comment]
also i don't think search bar color should change on hover

**Review by samarmohan — APPROVED** [review]

**Review by ajmnz — APPROVED** [review]
This will probably get wiped at some point, but it's fine for now

**Pull request merged**